images/runtime/php-fpm/8.2/8.2.Dockerfile

```diff
@@ -0,0 +1,250 @@
+ARG DEBIAN_FLAVOR
+# From https://github.com/docker-library/php.git
+FROM oryxdevmcr.azurecr.io/private/oryx/php-fpm-run-base-${DEBIAN_FLAVOR}
+ARG IMAGES_DIR=/tmp/oryx/images
+
+# do NOT merge this content with above line because the 
+# above line is shared across all php images
+# Install the Microsoft SQL Server PDO driver on supported versions only.
+#  - https://docs.microsoft.com/en-us/sql/connect/php/installation-tutorial-linux-mac
+#  - https://docs.microsoft.com/en-us/sql/connect/odbc/linux-mac/installing-the-microsoft-odbc-driver-for-sql-server
+RUN set -eux \
+	&& apt-get update \
+	&& apt-get install -y --no-install-recommends \
+		gnupg2 \
```

Comment:
I'm just curious: did we have to resolve any case of packages not supported on 8.2 yet, as it was just released? Or were all 8.1 packages also working with 8.2 automatically?

Reply:
All 8.1 packages worked with 8.2 as well automatically. We did _not_ have to resolve any packages.

Reply:
OK that's great. Thanks for confirming